Python/Product/TestAdapter.Executor/Config/PythonProjectSettings.cs

```diff
@@ -48,7 +48,7 @@ public class PythonProjectSettings : IEquatable<PythonProjectSettings> {
             EnableNativeCodeDebugging = nativeDebugging;
             SearchPath = new List<string>();
             Environment = new Dictionary<string, string>();
-            Sources = new List<string>();
+            TestContainerSources = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
I think it deserves an explanation here as to what the key and value are, and a bit why

Reply:
good point